splinter/driver/webdriver/remote.py

```diff
@@ -43,7 +43,7 @@ def __init__(
             # Combine user's desired capabilities with default
             caps.update(kwargs["desired_capabilities"])
 
-        kwargs["desired_capabilities"] = caps
+            kwargs["desired_capabilities"] = caps
```

Comment:
Isn't this a change of behavior? What's the impact of no longer setting desired_capabilities to `{}`?

Reply:
@fsouza `desired_capabilities` was deprecated; it was fully removed as part of selenium 4.10 so now you would get an invalid/unexpected keyword argument. Options that were passed as desired_capabilities are now handled with other method calls

Reply:
Right, but what's the impact of this change on code running with selenium <4.10?

Reply:
Would it make more sense to do a version check of the imported selenium lib and then either keep or delete `desired_capabilities` accordingly?

Reply:
Ohh I see that the parameter has been deprecated for ages. This PR looks good then.